ApiTests/Tests/API/APITests.cs

```diff
@@ -5,30 +5,33 @@ namespace TestProject.Tests.API
 {
     public class Tests
     {
+        private Pet pet;
+
         [SetUp]
         public void Setup()
         {
             //move test data creation here
+            pet = new Pet(420000, "Jake", "Alive");
         }
 
         [Test]
         public void PetTest()
         {
             //create a pet using post request
-            PetStoreApiUtils.PostPet(
-                new Pet(
-                    ConfigReader.GetNumericalTestDataValue("petId"),
-                    ConfigReader.GetTestDataValue("petName"),
-                    ConfigReader.GetTestDataValue("petStatus")));
+            PetStoreApiUtils.PostPet(pet);
 
             //validate that the name of the pet is as you passed in a previous step
+            Assert.That(PetStoreApiUtils.GetPetById(pet.Id).Name,Is.EqualTo(pet.Name));
```

Comment:
It's a good practice to add a message to your assertion. 
In case if it fails - system message will be 'Expected true but got false', so adding our own message we can help to speed up the investigation (there should be something short and informative, adding some context, for example - 'Pet name is not as expected')